### ISSUE TITLE
tensorboard 관련 질문입니다.

### ISSUE THREAD

**Issue by Yeerim** [MSWon/Sentimental-Analysis#1]
다른 텍스트파일로 코드를 따라해보았는데, 원인은 알수없지만 텐서보드를 열면 그래프는 그려져있는데, 한글이 아니라 각 점들이 숫자로 태깅되어있습니다ㅠ 혹시 해결방법을 아시나요...ㅠㅠ

WORD2VEC 의 경우에 most_similar 값도 잘 프린트되는걸로 봐서 러닝이 잘 된것같은데
텐서보드 부분이 말썽입니다ㅠㅠ

import gensim
import codecs
import os
import numpy as np

doc = open(r'C:\Users\Lab01\Desktop\JYR\jupyter\new\tweet_all_word.txt', 'r')
doc1 = doc.readlines()
total_word = []
for line in doc1:
    words = line.split(' ')
    final_word = []
    for word in words:
        final_word.append(word)
    total_word.append(final_word)

tokens = total_word
model = gensim.models.Word2Vec(size=300,sg = 1, alpha=0.025,min_alpha=0.025, seed=1234)
model.build_vocab(tokens)
    
for epoch in range(30):           
    model.train(tokens,model.corpus_count,epochs = model.iter)
    model.alpha -= 0.002
    model.min_alpha = model.alpha

model.save('Word2vec_tweet.model')

print (model.most_similar(positive=["기후변화"], topn=30))

import tensorflow as tf

max_size = len(model.wv.vocab)-1
w2v = np.zeros((max_size,model.layer1_size))

with codecs.open(r"C:\Users\Lab01\Desktop\JYR\jupyter\new\metadata2.tsv",'w+',encoding='utf-8') as file_metadata:
    for i,word in enumerate(model.wv.index2word[:max_size]):
        w2v[i] = model.wv[word]
        file_metadata.write(word + "\n")

from tensorflow.contrib.tensorboard.plugins import projector

sess = tf.InteractiveSession()
  
with tf.device("/cpu:0"):
    embedding = tf.Variable(w2v, trainable = False,  name = 'embedding')   

tf.global_variables_initializer().run() 

path = 'word2vec'

saver = tf.train.Saver()
writer = tf.summary.FileWriter(path, sess.graph)

config = projector.ProjectorConfig()
embed = config.embeddings.add()
embed.tensor_name = 'embedding'
embed.metadata_path = r'C:\Users\Lab01\Desktop\JYR\jupyter\new\metadata2.tsv'

projector.visualize_embeddings(writer, config)
saver.save(sess, path + '/model.ckpt' , global_step=max_size)
